src/main/scala/com/nitro/nmesos/sidecar/SidecarUtils.scala

```diff
@@ -86,29 +86,39 @@ object SidecarUtils {
       sidecarInfo: Seq[String],
       serviceName: String
   )(implicit fmt: Formatter): Boolean = {
-    if (containerInfo.sameElements(sidecarInfo)) {
+    val moreContainerInfos = containerInfo.diff(sidecarInfo) 
+    val moreSidecarInfos = sidecarInfo.diff(containerInfo)
+
+    if (moreContainerInfos.isEmpty && moreSidecarInfos.isEmpty) {
       fmt.println(
-        s""" ${fmt.Ok} Sidecar mapping for $serviceName match all containers running """
+        s""" ${fmt.Ok} Sidecar mapping for ${serviceName} match all containers running """
       )
-      sidecarInfo.foreach { info =>
-        fmt.info(s"\t\t$info")
+      containerInfo.foreach { info =>
+        fmt.info(s"\t\t${info}")
       }
       true
-    } else if (containerInfo.isEmpty) {
-      fmt.info("\tFound (in Sidecar, but not in mesos):")
-      sidecarInfo.foreach { info =>
-        fmt.info(s"\t\t$info")
+    } else if (moreContainerInfos.isEmpty && !moreSidecarInfos.isEmpty) {
```

Comment:
If we got more in sidecar, we are still good, but want to warn about what we found in sidecar (that is not in mesos).